server.js

```diff
@@ -91,6 +91,11 @@ exports.buildExpress = function(options) {
         res.end();
       });
     });
+    
+    if (req.body) {
+      console.log('writing body: ' + JSON.stringify(req.body));
+      mlReq.write(JSON.stringify(req.body));
+    }
 
     req.pipe(mlReq);
```

Comment:
Sure the mlReq.write is needed? Line 100 has `req.pipe(mlReq)` which should essentially pass the body through unchanged. Then again, perhaps there is some json parsing going on that messes things up.

Keep in mind demo-cat has been running for a long while, and we have not observed this so far..